conf/awsbatch.config

```diff
@@ -0,0 +1,25 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for AWSBatch
+ * -------------------------------------------------
+ * Imported under the 'aws' Nextflow profile in nextflow.config
+ * Defines reference genomes, using iGenome paths from s3
+ * Uses docker for software depedencies.
+ */
+
+docker {
+  enabled = true
+  fixOwnership = true
+  runOptions = "-u \$(id -u):\$(id -g)"
+}
+
```

Comment:
Docker options are ignored by he AWS Batch executor.

Reply:
Thanks @pditommaso ! I adjusted that :-)

Reply:
You should also remove `docker.enabled` :)

Reply:
Done!